test/integration/test_enterprise_management_v1.py

```diff
@@ -15,389 +14,265 @@
-
-    def test_16_create_standard_account(self):
-        self.__class__.standard_email = "aminttest+" + str(int(time.time() * 1000)) + "_" + str(math.floor(random.random() * 10000)) + "@mail.test.ibm.com"
-        jsonData = {
-            'owner_user_id': self.standard_email,
-            'owner_email': self.standard_email,
-            'owner_first_name': 'TEST',
-            'owner_last_name': 'TEST',
-            'owner_phone': '123456789',
-            'owner_company': 'IBM',
-            'country_code': 'USA',
-            'bluemix_subscriptions': [
-              {
-                'type': 'STANDARD',
-                'state': 'ACTIVE',
-                'part_number': 'COE-Trial',
-              },
-            ],
-            'ibmid': {
-              'password': 'password',
-              'question': 'question',
-              'answer': 'answer',
-            }
-        }
-        headers = {}
-        headers['content-type'] = 'application/json'
-        headers['authorization'] = self.service_token
-        url = self.am_host + "/coe/v2/accounts"
-        response = requests.post(url=url, headers=headers, json=jsonData)
-        resp = response.json()
-        self.__class__.standard_account_id = resp["id"]
-
-    def test_17_get_activation_code(self):
-        headers = {}
-        headers['content-type'] = 'application/json'
-        headers['authorization'] = self.service_token
-        url = self.am_host + "/v1/activation-codes/" + self.standard_email
-        time.sleep(20)
-        response = requests.get(url=url, headers=headers)
-        
-        resp = response.json()
-        x = resp["resources"]
-        y = x[0]
-        self.__class__.activation_token = y["id"]
-
-    def test_18_get_am_coe_v2_accounts_verify(self):
-        url = self.am_host + "/coe/v2/accounts/verify"
-        params = {
-            'token': self.activation_token,
-            'email': self.standard_email
-        }
-        response = requests.get(url=url, params=params)
-
-    def test_19_import_account_to_enterprise(self):
-        response = self.service.import_account_to_enterprise(
+        )
+
+        assert list_account_groups_response.get_status_code() == 200
+        list_account_groups_response = list_account_groups_response.get_result()
+        assert list_account_groups_response is not None
+
+    @needscredentials
+    def test_list_account_groups_with_pagination(self):
+
+        results_count = 0
+        doc_id = None
+
+        while True:
+            list_account_groups_response = self.enterprise_management_service.list_account_groups(
+                enterprise_id=self.enterprise_id,
+                limit=result_per_page,
+                next_docid=doc_id,
+            )
+
+            assert list_account_groups_response.get_status_code() == 200
+            list_account_groups_response = list_account_groups_response.get_result()
+            assert list_account_groups_response is not None
+
+            results_count += 1
```

Comment:
```suggestion
            results_count += list_account_groups_response.get('rows_count')
```
Yes, on the surface, this is equivalent to just incrementing the count by 1, but if we ever change limit, then you're innoculated :)